eth/stagedsync/stage_headers.go

```diff
@@ -324,9 +341,6 @@ func HeadersPOW(
 	var headerProgress uint64
 	var err error
 
-	if !useExternalTx {
```

Comment:
Why did you remove this?

Reply:
because it is unneded, it should have already removed it with `SpawnHeadersStage` but forgot